ci/steps/publish-npm.sh

```diff
@@ -74,22 +87,40 @@ main() {
   # We only need to run npm version for "development" and "staging".
   # This is because our release:prep script automatically bumps the version
   # in the package.json and we commit it as part of the release PR.
-  if [[ "$ENVIRONMENT" == "production" ]]; then
+  if [[ "$NPM_ENVIRONMENT" == "production" ]]; then
     NPM_VERSION="$VERSION"
+    # This means the npm version will be published as "stable"
+    # and installed when a user runs `yarn install code-server`
+    NPM_TAG="latest"
   else
+    COMMIT_SHA="$GITHUB_SHA"
     echo "Not a production environment"
-    echo "Found environment: $ENVIRONMENT"
+    echo "Found environment: $NPM_ENVIRONMENT"
     echo "Manually bumping npm version..."
 
-    if ! is_env_var_set "PR_NUMBER_AND_COMMIT_SHA"; then
-      echo "PR_NUMBER_AND_COMMIT_SHA is not set. This is needed for setting the npm version in non-production environments."
-      exit 1
+    if [[ "$NPM_ENVIRONMENT" == "beta" ]]; then
```

Comment:
I think this should say "staging". `beta` is a tag, not an `NPM_ENVIRONMENT`
```suggestion
    if [[ "$NPM_ENVIRONMENT" == "staging" ]]; then
```